swarm/pss/pss_test.go

```diff
@@ -68,6 +71,75 @@ func init() {
 	wapi = whisper.NewPublicWhisperAPI(w)
 }
 
+func TestCache(t *testing.T) {
```

Comment:
It isn't immediately obvious what is being testing here, a comment would be very useful.